pkg/tsdb/prometheus/time_series_query.go

```diff
@@ -128,7 +128,12 @@ func (s *Service) executeTimeSeriesQuery(ctx context.Context, req *backend.Query
 		return &result, err
 	}
 
-	return s.runQueries(ctx, client, queries)
+	fillNulls := true
+	if req.Headers["FromAlert"] == "true" {
```

Comment:
should this be handled on the frontend when the request is for a time series panel, or maybe by a new `Format` option in the query?

it seems like this might also be needed for the `Table` format. here's an example from this branch:
![image](https://user-images.githubusercontent.com/360020/151046111-9c0f306d-7b06-47f9-b730-b6c0e39add94.png)

vs `8.2.4`:
![image](https://user-images.githubusercontent.com/360020/151046252-9009f0b4-afd2-4309-a27f-ad54cff64585.png)